kalite/coachreports/static/js/coachreports/coach_reports/router.js

```diff
@@ -0,0 +1,39 @@
+CoachReportRouter = Backbone.Router.extend({
+    initialize: function(options) {
+        _.bindAll(this);
+        this.state_model = new StateModel();
+        this.listenTo(this.state_model, "change", this.set_url);
```

Comment:
Identifier 'state_model' is not in camel case.<br>Identifier 'set_url' is not in camel case.